packages/manager/apps/telecom/src/app/telecom/pack/migration/pack-migration.controller.js

```diff
@@ -0,0 +1,45 @@
+import get from 'lodash/get';
+import head from 'lodash/head';
+
+export default class TelecomPackMigrationCtrl {
+  /* @ngInject */
+  constructor($translate, TucPackMigrationProcess, TucToast) {
+    this.$translate = $translate;
+    this.TucPackMigrationProcess = TucPackMigrationProcess;
+    this.TucToast = TucToast;
+  }
+
+  /*= =====================================
+  =            INITIALIZATION            =
+  ====================================== */
+
+  $onInit() {
+    this.loading = {
+      init: false,
+    };
+    this.process = null;
+
+    this.loading.init = true;
+    this.process = this.TucPackMigrationProcess.init(this.packName);
+
+    return this.TucPackMigrationProcess.checkForPendingMigration()
+      .then((pendingTasks) => {
+        if (pendingTasks && pendingTasks.length === 1) {
```

Comment:
isn't it possible that there is multiple pending task and one is a migration ?

Reply:
no it's not